index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
             </li>             
      </ul>               
     </section>
+    <section class="works">
+        <div class="title">
+           <h2>My Recent Works</h2>
+           <div class="line"></div>
```

Comment:
- [ ] For some reason, the horizontal line that is supposed to be in this section that is present in the Figma design cannot be seen in your design. Kindly look into why the horizontal line is not showing.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/185756032-380c46ec-ca8e-45c0-8143-e0a9da54e70b.png)

**How it is supposed to be according to the Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185756050-47bc1b90-334d-4ef0-b826-e64228a6db76.png)